### DISCUSSION TITLE
golang style jetstream examples

### DISCUSSION THREAD

**Discussion by guojianyu** [nats-io/nats.go#942] [Ideas]
## Feature Request

More golang examples are provided to quickly use JetStream

**Comment by jnmoyne** [comment]
Did you see the updated developers guide in the documentation? Including the new 'anatomy of a NATS application' which is meant to help new developers get going quickly and has a section on how to use JetStream (https://docs.nats.io/using-nats/developer/anatomy#streaming-with-jetstream) which itself points to a new JetStream section of the developer's guide which has code sample in all the major languages including Golang (https://docs.nats.io/using-nats/developer/develop_jetstream).
